coursedata/adventures/de.yaml

```
@@ -1304,12 +1397,14 @@ adventures:
                     ## Restaurant
                      In level 1 you can make your own virtual restaurant and take your guests' orders.
 
+                example_code: |
                     ## Example Hedy code
+
                       ```
                      print Welcome to Hedy's restaurant 🍟
                      ask What would you like to order?
-                     echo So you would like to order 
-                     print Thank you for your order!
+                     echo So you would like to order
+                     print Thanks you for your order!
```

Let's not regress w.r.t. #1848
```suggestion
                     print Thank you for your order!
```

---

See above.

---

Same here, actually... It looks like the regression has taken place already. In addition, this block is ill-formatted. The code block is indented one space too far. I'll hopefully keep it on the list for when I reach it.

---

> It looks like the regression has taken place already.

Indeed. Probably happened when resolving merge conflicts in bfe8bcb40ccbebb5d3a81f35a5059b1384afce47 within #1838.

---

> Let's not regress w.r.t. #1848

This change is in main, so it will be fixed automatically when we update this branch from main. No need to keep this PR up to date with all others.